appeals/web/src/server/lib/nunjucks-template-builders/page-component-rendering.js

```diff
@@ -36,7 +36,7 @@ export const preRenderPageComponents = (pageComponents, recursions = 0, maximumR
 					for (const itemKey of itemKeys) {
 						const itemProperty = item[itemKey];
 
-						if (!isObjectLiteral(itemProperty)) {
+						if (!isObjectLiteral(itemProperty) || item?.value?.html?.length) {
```

Comment:
This stops this function overwritting and HTML property that has already been written as a string. For example here where we've used `buildHtmUnorderedList` to create the attachments list, this previously got destroyed.